tests/acceptance/features/webUIAdminSettings/adminGeneralSettings.feature

```diff
@@ -8,7 +8,7 @@ Feature: admin general settings
     Given the administrator has changed their own email address to "admin@owncloud.com"
     And the administrator has browsed to the admin general settings page
 
-  @smokeTest @skip @yetToImplement
+  @smokeTest @issue-5015
```

Comment:
To make it uniform, we can either remove the smoke test tag from here or keep the existing tag in `adminAppsSettings.feature`